### ISSUE TITLE
Incorrect equivalent latitude calculation

### ISSUE THREAD

**Issue by kuchaale** [pankajkarman/pyvortex#2]
Hi @pankajkarman, thanks for the `pyvortex` module - very helpful. However, I may have found a bug in the equivalent latitude calculation. See my gist [here](https://gist.github.com/kuchaale/8a58ce069fc4658da410c9e05f3a3bc0). Therein I use PV and ZW from ERA-Interim at 600 K and two different time steps (2008-12-03T12, 2009-01-05T00). The equivalent latitude is incorrect for the latter time step. Once I modify [`_eqvlat`](https://github.com/pankajkarman/pyvortex/blob/master/pyvortex/pyvortex.py#L116) after [`hn2016_falwa.basis.eqvlat`](https://github.com/csyhuang/hn2016_falwa/blob/master/hn2016_falwa/basis.py#L5) and consequently `get_edge`, I produce more reasonable results. Looking forward to your response.

**Comment by pankajkarman** [comment]
Hi @kuchaale, Thank you very much for your interest in this package and I am glad you found it useful. Your concern is quite valid. The issue seems to arise from the formula used for calculation of equivalent latitude used. While pyvortex uses equation from [Anel et al.](https://dx.plos.org/10.1371/journal.pone.0072970) for estimation of EqL i.e.

![allen2](https://user-images.githubusercontent.com/34066356/143601707-4722aa8c-8008-46e1-ab2a-a19aaaf73c84.png)

[hn2016_falwa.basis.eqvlat](https://github.com/csyhuang/hn2016_falwa/blob/master/hn2016_falwa/basis.py#L5) uses equation from [Allen et al. ](http://journals.ametsoc.org/doi/10.1175/1520-0469(2003)060%3C0287:TELADT%3E2.0.CO;2) for the same.

![allen1](https://user-images.githubusercontent.com/34066356/143601694-c18fd2d4-3739-4b0f-9e1c-f8f367259efe.png)

While these seemed to be working ok till now, but if this issue is indeed due to the equation mentioned above, I'll change it to align with correct result. Can please check and confirm?

**Comment by kuchaale** [comment]
You're right.  [Anel et al.](https://journals.plos.org/plosone/article?id=10.1371/journal.pone.0072970#pone.0072970-Allen1) actually used the formula from [Lary et al.](https://davidlary.info/wp-content/uploads/2012/08/Q.-J.-R.-Meteorol.-Soc.-1995-121-pp.-187-210.pdf) (see eq. 4 in the appendix). They even provide their IDL codes [here](https://github.com/eqlat/roi/blob/master/eqlat_roi_0.1.2.pro). Maybe @eqlat can explain the formula difference?

**Comment by JAAxel** [comment]
Hi everyone,
Thanks @kuchaale for letting me know about this issue. I have taken a quick look into the bug report, but I have no time today to check if this is an issue with pyvortex or our original code.
Could you provide more detail about where you think that is the bug? Cause?
I do not know if it is the case here, but I recall subtle differences between the original Lary et al. and the posterior version by Douglas Allen.

By the way, nice to see that you are using this. If you have based your routines or computation on the ideas or software in our paper, please, cite it. This is encouraging, lets us know about the community using it, and helps us secure funding for our research work.

**Comment by kuchaale** [comment]
Hi @JAAxel, the issue is related to [pyvortex](https://github.com/pankajkarman/pyvortex). Your paper is mentioned as a reference for one way how to calculate the equivalent latitude. References quite differ on this matter. Additionally, [Nakamura (1995)](http://journals.ametsoc.org/doi/10.1175/1520-0469(1995)052%3C2096:MLMDOT%3E2.0.CO;2) and [Norton (1994)](https://journals.ametsoc.org/view/journals/atsc/51/4/1520-0469_1994_051_0654_brwiam_2_0_co_2.xml?tab_body=pdf) formulate the equivalent latitude in agreement with Lary et al and Anel et al (see Eqs. 2.11 and A1, respectively). The formulas are just inverse to each other but apparently, it matters.

Your IDL routines may be useful but internal IDL functions such as `ComputeGeometry` and `contour` require IDL license, I guess.

**Comment by pankajkarman** [comment]
fixed
